app/views/workarea/storefront/pages/_robots.text.erb

```diff
@@ -0,0 +1 @@
+Disallow: /wish_lists
```

Comment:
Can you unique this file name a bit more? Seems a little common, like if a build wanted to append some robots stuff, they could use this name.

Reply:
You got it!